pkg/report/writer.go

```diff
@@ -52,6 +52,12 @@ func WriteResults(format string, output io.Writer, results Results, outputTempla
 				}
 				return escaped.String()
 			},
+			"endWithPeriod": func(input string) string {
+				if !strings.HasSuffix(input, ".") {
+					input += "."
+				}
+				return input
+			},
```

Comment:
This seems untested. You can add a unit test here https://github.com/aquasecurity/trivy/blob/e7e3bb45ce59b736e29a88cbe68a6970db241bdf/pkg/report/writer_test.go#L199-L294

Reply:
@simar7 Done